Jenkinsfile

```diff
@@ -47,9 +47,19 @@ pipeline {
               sh "cp \$ethereum_password configs/eth/ethereum_password_local.properties"
             }
             sh "./gradlew dependencies"
+            sh "./gradlew dokka"
             sh "./gradlew test --info"
             sh "./gradlew compileIntegrationTestKotlin --info"
             sh "./gradlew integrationTest --info"
+            sh "./gradlew codeCoverageReport --info"
+            sh "./gradlew dokka --info"
```

Comment:
What are the differences between similar command above and this one?

Reply:
1 - code coverage
2 - code documentation